js/main.js

```diff
@@ -25,6 +27,14 @@ function isGoogleChrome() {
   return false;
 }
 
+async function isBrave() {
+  if (await navigator.brave.isBrave()) {

```

Comment:
`navigator.brave` is undefined probably in other browsers? (e.g. my firefox) so there this fould fail with an error. 

should this be something like:
```
if (!navigator.brave) { return false; }
const isBrave = await navigator.brave.isBrave();
return isBrave;
```

Reply:
I tested this with Firefox and it displays the correct button in this case. I thought I had accounted for other browsers being false by putting the ```else { return false; }``` statement in there. If this conditional at the top of the file returns false, the code will continue to run onto the next line. I haven't tested it with Chrome though, do you have Chrome that you could check with?

Reply:
@bumi